benchmarks/pipe.py

```diff
@@ -681,14 +580,18 @@ def bench_mpi(args):
 parser.add_argument(
     "--no-pipelined-backward", dest="pipelined_backward", action="store_false", help="Pipelined backward pass"
 )
+parser.add_argument("--use_synthetic_data", default=True, help="Uses synthetic data to run benchmarks.")
```

Comment:
the help msg doesn't seem to match the actual semantics in the code?

Reply:
I was thinking of the eventual state :) I modified it to reflect that we will use synthetic data for only a sample training run.